include/git2/remote.h

```diff
@@ -41,6 +41,16 @@ GIT_EXTERN(int) git_remote_create(
 		const char *name,
 		const char *url);
 
+/**
+ * Remote creation options flags
+ */
+typedef enum {
+	GIT_REMOTE_CREATE_DEFAULT = 0,
```

Comment:
Why is this flag necessary? We don't have default flags for all the other option enums, do we?

Reply:
Fixed.